deploy/olm-catalog/bundle/manifests/crane-operator.v99.0.0.clusterserviceversion.yaml

```diff
@@ -229,18 +229,18 @@ metadata:
     operatorframework.io/suggested-namespace: openshift-migration
 spec:
   skips:
-  - konveyor-operator.v1.4.2
-  - konveyor-operator.v1.4.1
-  - konveyor-operator.v1.4.0
-  - konveyor-operator.v1.3.2
-  - konveyor-operator.v1.3.1
-  - konveyor-operator.v1.3.0
-  - konveyor-operator.v1.2.5
-  - konveyor-operator.v1.2.4
-  - konveyor-operator.v1.2.3
-  - konveyor-operator.v1.2.2
-  - konveyor-operator.v1.2.1
-  - konveyor-operator.v1.2.0
+  - crane-operator.v1.4.2
+  - crane-operator.v1.4.1
+  - crane-operator.v1.4.0
+  - crane-operator.v1.3.2
+  - crane-operator.v1.3.1
+  - crane-operator.v1.3.0
+  - crane-operator.v1.2.5
+  - crane-operator.v1.2.4
+  - crane-operator.v1.2.3
+  - crane-operator.v1.2.2
+  - crane-operator.v1.2.1
+  - crane-operator.v1.2.0
```

Comment:
Is this the correct thing to do here?

Reply:
If I recall correctly, we decided not to convert old versions, so just remove them all. We'll have to add 1.4.3 or whatever here as we move forward. But once we rename the operator there will be no upgrade path from konveyor -> crane so there's no point in keeping these.

Reply:
So far the rest of the changes look sane as well.

When we go to push the metadata we'll want to clear out delete:
https://quay.io/application/konveyor/konveyor-operator

I also didn't write the metadata push playbook with bootstrapping a new application so there's a good chance the version it writes will be garbage and it'll fail, but we can deal with that later.

Reply:
We might also want to re-bootstrap the bundle/index image since we'll have all the old konveyor versions in there. I think this would make the most sense going forward though I think we can keep both konveyor and crane in there for awhile if we want...

Reply:
`Remove skips:` https://github.com/konveyor/mig-operator/pull/612/commits/2864c57beef57f0f432d9f21c70febecb2f57c4d